lib/index.ts

```diff
@@ -13,6 +13,7 @@ import { BlueprintLayout, BlueprintObject, ContractObject } from './types';
 import Contract from './contract';
 import Blueprint from './blueprint';
 import Universe from './universe';
+import ObjectSet from './object-set';
```

Comment:
This can also be done as
```suggestion
export { default as ObjectSet } from './object-set';
```
to avoid importing purely to re-export